packages/engine/src/networking/serialization/Utils.ts

```diff
@@ -81,14 +81,16 @@ export const FLOAT_PRECISION_MULT = 1000
 // giving more precision with the same number of bits.
 export const QUAT_MAX_RANGE = 1 / Math.sqrt(2)
 
+export const VEC3_MAX_RANGE = 1 / 32
```

Comment:
Why is this 1/32?

Reply:
Gaffer suggested that a speed of 32 meters per-second is a good starting point. I tested it out using our physics simulation test system and everything was working as expected.

Reply:
this seems fine for now, we can see if puttclub ends up having a ball speed faster than this and we can adjust as needed

Reply:
We should rename this constant if it's strictly related to Velocity. Vec3_Max_Range is ambiguous